misc/scripts/create-change-note.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+# Creates a change note and opens it in VSCode for editing.
+
+# Expects to receive the following arguments:
+# - What language the change note is for
+# - The name of the change note (in kebab-case)
+# - The category of the change.
+
+# The change note will be created in the `{language}/ql/lib/change-notes` directory.
```

Comment:
What about query changes? Which should go in `{language}/ql/src/change-notes`?

Reply:
☝️

Reply:
Honestly, the reason I targeted `lib` is that in my experience that's where the vast, _vast_ majority of changes take place. I figured if you wanted to create a query change note, you could just create it in `lib` and move it over afterwards. However... (continued in a reply to your other comment)